test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -80,8 +80,8 @@ public void ParentItems_SetsItemsProperty()
 
             // Assert
             Assert.Same(content1, content2);
-            Assert.Equal(expectedContent, content1);
-            Assert.Equal(expectedContent, content2);
+            Assert.Equal(expectedContent, content1.ToString());
+            Assert.Equal(expectedContent, content2.ToString());
```

Comment:
not sure about overriding `ToString()` but should definitely use `Get()` ourselves